readme.md

```diff
@@ -42,6 +42,48 @@ that POs have been given the authority and means necessary to complete them. :mu
 
 * product backlog: a prioritised features list for the product found on github.
 
+See: https://github.com/dwyl/process-handbook/issues/61, linked to #5
+
+## Template for making an issue
+
+**The 3 components you should include when creating an issue are:**
+
+1. A User story
+2. Acceptance Criteria
+3. A screenshot of a mock-up or a bug on the site (where possible)
+
+Followed by... any other useful/ relevant information for completing the issue.
+
+### User Stories
+
+A user story is a phrase which describes the what, why and who is it for of each
+issue. To write one complete the following three starters:
+
+**As...** a team member - **WHO**
+**I want to...** see a user story on every issue I read  - **WHAT**
+**so that...** I know the context in which the issue is set to help me understand
+it better - **WHY**
+
+**Here's another example:**
+
+**As** someone in need of a bin
+**I want to** be able to filter on the online shopping site by 'lid type'
+**So that** I can find a bin that closes securely to stop mice getting into my
+bin 🐭 ❌ 🗑 🧀
+
+### Acceptance Criteria
+
+Acceptance criteria are checkboxes that a developer knows they need to complete
```

Comment:
This isn't particularly easy for a PO to understand, we'll definitely need to point to examples.